tests/unit/preprocessor/_time/test_time.py

```diff
@@ -36,7 +49,6 @@ def setUp(self):
     def test_get_january(self):
         """Test january extraction"""
         sliced = extract_month(self.cube, 1)
-        print(sliced)
```

Comment:
Put the `print` functions back, they help debugging if the test fails.

Reply:
ok since the print call will print only if it fails

Reply:
done